test/chromedriver_utils.dart

```diff
@@ -51,7 +51,7 @@ Future<wd.WebDriver> createWebDriver() async {
       spec: wd.WebDriverSpec.JsonWire,
       desired: capabilities,
       uri: Uri.parse(
-          'http://127.0.0.1:$chromeDriverPort/$chromeDriverUrlBase/'));
+          'http://localhost:$chromeDriverPort/$chromeDriverUrlBase/'));
```

Comment:
just curious, why was this change needed?

Reply:
Looks like github actions refuses connections on 127.0.0.1 https://github.com/Workiva/webdev_proxy/actions/runs/4537255878/jobs/7994904946
```
00:00 +0 -1: test/sse_proxy_handler_test.dart: Can round trip messages [E]
  SocketException: Connection refused (OS Error: Connection refused, errno = 111), address = 127.0.0.1, port = 37522
  dart:_http                                                        _HttpClient.postUrl
  package:webdriver/src/request/async_io_request_client.dart 30:36  AsyncIoRequestClient.sendRaw
  ===== asynchronous gap ===========================
  dart:_http                                                        _HttpClient.postUrl
  package:webdriver/src/request/async_io_request_client.dart 30:36  AsyncIoRequestClient.sendRaw
  ===== asynchronous gap ===========================
  dart:async                                                        _CustomZone.registerUnaryCallback
  package:webdriver/src/request/async_io_request_client.dart 22:5   AsyncIoRequestClient.sendRaw
  package:webdriver/src/common/request_client.dart 96:39            AsyncRequestClient.send
  package:webdriver/async_core.dart 62:32                           createDriver
  package:webdriver/async_io.dart 41:10                             createDriver
  test/chromedriver_utils.dart 50:30                                createWebDriver
  ===== asynchronous gap ===========================
  dart:async                                                        _CustomZone.registerUnaryCallback
  test/chromedriver_utils.dart 40:[21](https://github.com/Workiva/webdev_proxy/actions/runs/4537255878/jobs/7994904946#step:7:22)                                createWebDriver
  test/sse_proxy_handler_test.dart 47:[23](https://github.com/Workiva/webdev_proxy/actions/runs/4537255878/jobs/7994904946#step:7:24)                            main.<fn>
  ===== asynchronous gap ===========================
  dart:async                                                        _CustomZone.registerUnaryCallback
  test/sse_proxy_handler_test.dart 36:45                            main.<fn>
```

Reply:
So did we eventually determine that this was a red herring?

Reply:
No turns out that this seems to still cause issues unless it's changed to localhost